Formula/arttime.rb

```diff
@@ -0,0 +1,26 @@
+class Arttime < Formula
+  desc "Clock, timer, time manager and ASCII+ text-art viewer for the terminal"
+  homepage "https://github.com/poetaman/arttime"
+  url "https://github.com/poetaman/arttime/archive/refs/tags/v2.0.0.tar.gz"
+  sha256 "385c8ddf39653ab52c1c1ea8edca14c19cb3eb05c8d1e6627201ccb2cc191755"
+  license "GPL-3.0-only"
+  head "https://github.com/poetaman/arttime.git", branch: "main"
+
+  on_linux do
+    depends_on "diffutils"
+    depends_on "less"
+    depends_on "libnotify"
+    depends_on "vorbis-tools"
+    depends_on "zsh"
+  end
+
+  def install
+    ENV["TERM"]="xterm"
+    system "./install.sh", "--noupdaterc", "--prefix", prefix.to_s, "--zcompdir", "#{share}/zsh/functions"
```

Comment:
```suggestion
    system "./install.sh", "--noupdaterc", "--prefix", prefix.to_s, "--zcompdir", zsh_completion
```

Reply:
Since this is for installing shell-completion, let's use zsh_completion macro. cc @ZhongRuoyu

Reply:
@chenrui333 @ZhongRuoyu In my local test, that works. Thanks!

Reply:
Yeah, in order to make completions work, zsh completions need to be put into `zsh_completion`. I pushed a new commit 0405cfb4e709e9dbc01de31396e5f93ebe7cf7a0 to address this.